backend/eventd/eventd.go

```diff
@@ -230,16 +241,15 @@ func (e *Eventd) handleMessage(msg interface{}) error {
 			return err
 		}
 		return e.handleUpdate(event)
-	} else {
-		// The check TTL has been disabled, there is no longer a need to track it
-		if err := switches.Bury(context.TODO(), switchKey); err != nil {
-			// It's better to publish the event even if this fails, so
-			// don't return the error here.
-			logger.WithError(err).Error("error burying switch")
-		}
+	}
+	// The check TTL has been disabled, there is no longer a need to track it
+	if err := switches.Bury(context.TODO(), switchKey); err != nil {
```

Comment:
Can you rebase your work on top of Simon's bugfix PR (or merge it in)? It changes how this section of the code works.

Reply:
Hah I was just about to merge this in and review the bugfix. One has to be merged in first and one has to be rebased, right? I suppose I could wait for his to be merged but I really don't see a difference..

Reply:
True. I'll leave it to you and Simon :)

Reply:
me waiting for @palourde to merge https://github.com/sensu/sensu-go/pull/2950

![giphy-1](https://user-images.githubusercontent.com/10634078/57725715-a0d8e980-7642-11e9-8e17-a29e7c77500b.gif)